gprofiler/log.py

```diff
@@ -189,6 +190,12 @@ def format(self, record: LogRecord) -> str:
         return formatted
 
 
+class GProfilerFormatter(ExtraFormatter):
+    # Patch formatTime to be GMT (UTC) for all formatters,
+    # see https://docs.python.org/3/library/logging.html?highlight=formattime#logging.Formatter.formatTime
+    converter = time.gmtime
```

Comment:
Hmm just add in `ExtraFormatter` no? It's already specific to gProfiler, why another `GProfilerFormatter`.

If you wanna write the "pretty thing" then I'd write a `UTCFormatter` class like this, and use it as a mixin in:
```
class GProfilerFormatter(UTCFormatter, ExtracFormatter): pass
```

Reply:
Also, pls rename `_ExtraFormatter` if it's not to be used directly anymore (only `GProfilerFormatter` is)

Reply:
Hmm, ok going with the mixin